vector/src/main/java/im/vector/activity/LoginActivity.java

```diff
@@ -620,6 +641,16 @@ private void fallbackToLoginMode() {
         refreshDisplay();
     }
 
+    /**
+     * Cancel the current mode to switch to the start one.
+     * It should restore the start UI
+     */
+    private void fallbackToStartMode() {
+        enableLoadingScreen(false);
```

Comment:
I think we should duplicate here the same code as `fallbackToLoginMode``

````
// display the main layout
         mMainLayout.setVisibility(View.VISIBLE);	         mMainLayout.setVisibility(View.VISIBLE);
 	 
         // cancel the registration flow	         // cancel the registration flow
         cancelEmailPolling();	         cancelEmailPolling();
         mEmailValidationExtraParams = null;	         mEmailValidationExtraParams = null;
         mRegistrationResponse = null;	         mRegistrationResponse = null;
         showMainLayout();	         showMainLayout();
````
In order to reset a potential registration in progress, do you agree?